src/components/index.ts

```diff
@@ -31,3 +31,5 @@ export { TabActions, TabContent, Tabs, TabsWithContent } from "./Tabs";
 export type { RouteTab, RouteTabWithContent, Tab, TabWithContent } from "./Tabs";
 export * from "./Tag";
 export * from "./Tooltip";
+export * from "./Accordion";
```

Comment:
If I put this at the top of the file (in alpbabetical order), it throws an export error, probably because it's using `Icon` that hasn't been exported yet

Reply:
Ah, maybe. You could always import the Icon from the Icon file directly vs from `src/index`.